### DISCUSSION TITLE
Support for Mousetrap Global Bind

### DISCUSSION THREAD

**Discussion by jarrodpayne** [reaviz/reakeys#15] [Ideas]
Sometimes, we need to register a keyboard shortcut that applies even if any other input is focused. My understanding is this is only possible using [Mousetrap's Global Bind plugin](https://github.com/ccampbell/mousetrap/tree/master/plugins/global-bind).

Is this something you would consider supporting?

**Comment by amcdnl** [comment]
@paynecodes - At the moment it does not support that but it would not be difficult to add. I'm thinking just pass a `global` flag and then bind based on that.
